pkg/apis/config/v1alpha1/resmgr/policy/topologyaware/config.go

```diff
@@ -77,4 +101,12 @@ type Config struct {
 	// of it.
 	// +kubebuilder:validation:Required
 	ReservedResources Constraints `json:"reservedResources"`
+	// DefaultCPUPriority (high, normal, low, none)
+	// This parameter is passed to CPU allocator when allocating CPUs exclusively.
+	// If a container is not annotated otherwise, this is the requested priority
+	// from the allocator.
+	// +kubebuilder:validation:Enum=high;normal;low;none
+	// +kubebuilder:default=none
+	// +kubebuilder:validation:Format:string
+	DefaultCPUPriority CPUPriority `json:"defaultCPUPriority,omitempty"`
```

Comment:
nit: This field can have 4 different values out of which `none`  is considered as the default. Should not it rather be called `CPUPriority`?

Reply:
But it is not the CPU priority. It is the default CPU priority, which is used if the user has not annotated the container otherwise (and if the policy does not decide otherwise during its internal classification of the container/workload). But at least I need to update the comment to state this in more clear terms.

Reply:
Updated the description to better reflect the above description.